packages/core/src/index.ts

```diff
@@ -1,6 +1,10 @@
 export { prepareCli } from './cli/prepare';
 export { runCli } from './cli/commands';
-export { defineConfig, loadConfig } from './config';
+export {
+  defineConfig,
+  loadConfig,
+  composeCreateRsbuildConfig as _composeCreateRsbuildConfig,
```

Comment:
```suggestion
  composeCreateRsbuildConfig as unstable_composeCreateRsbuildConfig,
```

I'd prefer prefix with `unstable_` just like React (https://github.com/search?q=repo%3Afacebook%2Freact%20unstable&type=code). Which can warn user in a more clear way.